src/contexts/UserContext.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from 'react'
+
+const Context = React.createContext({})
+
+export function UserContextProvider ({children}) {
+  const [token, setToken] = useState(
+    () => JSON.parse(window.localStorage.getItem('token'))
```

Comment:
create a constant for the token! other thing to keep in mind is that using localStorage is not good, the worst case possible is to keep the user information in the storage, a way to solve this is to have a BE that informs if the user token expires, and then redirects if the reponse of the token was 400, just to keep in mind

Reply:
What I am trying is that when I reload the page, it does not end my session and reading some documents suggested this approach. So do you think I should better delete this?

How can we then store these values to make requests to the BE?